src/references/mobile_device_preview.md

```diff
@@ -0,0 +1,91 @@
+    The device preview only works in editor and local preview mode. Multiplayer preview mode is currently not supported.
+
+    Only landscape mode is supported for now.
+
+![!Device Preview Options](../img/MobileDevicePreview/options.png){: .center loading="lazy" }
+
+#### Device frame
+
+When enabling the **Device Frame** option, the preview area will disable the device frame. In addition to simulating the dimensions of mobile devices, it will also reflect the notch on the display if supported by the device selected.
+
+![Device Frame](../img/MobileDevicePreview/withframe.png)
+![Device Frame](../img/MobileDevicePreview/noframe.png)
+*Device Frame Enabled (left), Device Frame Disabled (right)*
+{: .image-cluster}
+
+#### Flip Notch
+
+Using the **Flip Notch** option will toggle the notch between the left and the right side of the mobile device. Not all mobile devices have this notch, but it is important to test and ensure parts of the UI are not obstructed by the notch.
+
+![!Notch](../img/MobileDevicePreview/notch.png){: .center loading="lazy" }
+
+#### Show Safe Zone
+
+By enabling the **Safe Zone** option, the safe zone of the selected device will have the safe zone area displayed where all UI elements will be visible.
+
+![!Safe Zone](../img/MobileDevicePreview/safezone.png){: .center loading="lazy" }
+
+#### Devices
+
+The **Devices** dropdown allows creators to select a specific device to preview.
+
+![!Devices](../img/MobileDevicePreview/devices.png){: .center loading="lazy" }
+
+## Device Preview UI
+
+The device preview UI changes depending on where the player touches the screen. For example, when the player touches the left half of the screen, a virtual joystick will appear. The right half of the screen controls the camera's movement when touched.
+
+![!Enabled](../img/MobileDevicePreview/enabled.png){: .center loading="lazy" }
+
+### Default Mobile UI
+
+The default mobile UI has controls that give a player quick access to things such as Mounts, Emotes, and Settings.
+
+| Mobile UI Icon | Description |
+| -------------- | ----------- |
+| ![Core Menu](../img/MobileDevicePreview/UI/Core.png) | Minimizes the current game and brings up the Core game browser. |
+| ![Player Emotes](../img/MobileDevicePreview/UI/Emotes.png) | Opens the emote window for the player to select an emote to play. |
+| ![Player Mounts](../img/MobileDevicePreview/UI/Mounts.png) | Opens the mount window for the player to select a mount to ride. |
+| ![Settings](../img/MobileDevicePreview/UI/Settings.png) | Opens up the settings window for the player. |
+
+## Performance Limits
+
+Performance limits track the objects in the project Hierarchy and show the limit numbers for the object count, networked object, and terrain complexity. Creators should try to keep all bars below the limits to give players the best game performance. Reaching any limit will result in a warning, but currently does not block you from publishing.
```

Comment:
I have not tested yet, but there's a new option to publish for IoS that may restrict creators from publishing if the object counts are overcap.

Reply:
Will test this out on Monday and let you know. If so, will add to the ref.

Reply:
Looks like it gives a warning but allows for it to be published. 
![image](https://user-images.githubusercontent.com/93150109/162546641-b7fe22aa-94b3-4239-abb8-afd88d1cf8d8.png)